projects/addon-commerce/enums/currency.ts

```diff
@@ -149,6 +149,7 @@ export const enum TuiCurrency {
      * @description Alphabetic code: HKD
      * @description Countries and territories: Hong Kong
      */
+    // eslint-disable-next-line @typescript-eslint/naming-convention
     HongKong_dollar = 'HKD',
```

Comment:
lol)

Let's make it as deprecated ?
and create new one
```ts
HongKongDollar = 'HKD',
```

\+ write migration (or add task to https://github.com/Tinkoff/taiga-ui/issues/1764)

Reply:
fixed, and add task to https://github.com/Tinkoff/taiga-ui/issues/1764